jujugui/static/gui/src/app/app.js

```diff
@@ -2634,6 +2636,36 @@ YUI.add('juju-gui', function(Y) {
       maasContainer.show();
     },
 
+    // TODO: Move to views/utils.js.
+    showSharing: function(show = true) {
```

Comment:
It's a little weird that a method called `showSharing` can take an argument to do the exact opposite of the name :)

What about `sharingVisibility`?